dockerfiles/theia/README.md

```diff
@@ -1,33 +1,101 @@
 # How to Build Theia Image
 
 ## Build image manually using build scripts
-To build Theia image manually You could use build.sh scripts.
-Dockerfile uses multi-stage build feature, so first of all You need build builder image -
-theia-dev:
 
-Example:
+To build all the images you can easily run [build.sh](../../build.sh) script located in the repository root.
 
-```shell
-$ ../theia-dev/build.sh
+CI for checking any pull request in this repository runs this script with `--pr` directive
+```bash
+`build.sh --pr`.
+```
+
+**Note**: `--pr` will build only [limited set](../../docker_image_build.include) of docker images.	
+
+To publish docker images use `--push` directive
```

Comment:
where will these image be pushed ? is it only used by CI ? if yes could we specify that this is not intended to be used by the developer but CI ?
Also, CI related scripts should should not talk about it too much as they won't be used by the user anyway ...

Reply:
Let's remove this part at all. It is not used by the enduser, only by CI. So, it does not make a sense.

Reply:
ok